### DISCUSSION TITLE
🧰 本库使用指南

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#27] [主发布站的评论系统]
# 🧰 本库使用指南

[!INFO] 提示
本库不搞任务管理、美化排版等。尽量聚焦文本内容本身，使用起来相对简单。即便如此，若您看完本页内容后，依旧遇到了使用上的疑问，欢迎来交流讨论： Discussions · oldwinter/knowledge-garden · GitHub
 阅读小贴士：[[如何阅读由双链笔记组成的文章]]
兼容性说明：[[本库在不同操作系统下兼容性的说明]]
本库使用指南将围绕以下3个方面展开：
 如何高效使用obsidian软件本身。 如何构建个人的第二大脑/知识库。 如何将自己的数字花园/知识库发布分享出来。  Obsidian：使用篇 Obsidian从入门到放弃  [[众多双链笔记中我选obsidian的核心原因]] [[本库推荐的obsidian入门路线及资源]]   看了上面这些入门内容，也许已经一个月过去了😅。入门后才建议继续看本库其他obsidian相关内容，否则可能会浪费你的时间⏰。

https://garden.oldwinter.top/-%E6%9C%AC%E5%BA%93%E4%BD%BF%E7%94%A8%E6%8C%87%E5%8D%97

**Comment by Halecoder** [comment]
大佬，能不能讲一下你是如何做一篇笔记的，根据你所提到的笔记法，比如在哪创建笔记，如何选择编写什么样的tags便于管理,还有使用dataview搞MOC,TOC这些，我做了很多笔记，但是太混乱了，多谢

**Reply by oldwinter** [reply]
我的回答恐怕要让你失望。我现在还是缺少成片的学习时间，所以对我而言最适合的就是dailynote工作流，随时记录一小篇内容，并且看是否能和其他知识link起来。而你现在应该是学生时代？那就给成主题的一些书，课程的笔记建立一个MOC，然后在MOC里面建立相关知识点呗。
我也还在持续摸索，自己没用熟练的东西不敢乱讲误人子弟，哈哈哈，所以比如tag怎么用，dataview怎么用，你都自行探索吧。
我另一个观点：混乱不见得是坏事。为了所谓整齐，而去花大量时间整理，不见得划算，因为后期需要用到的时候，都是用搜索，而不是逐层去翻文件夹。

**Reply by Halecoder** [reply]
谢谢作者回复，我觉得还是先记笔记为主，只要产出内容，后面慢慢整理复习，感谢你优秀多的开源与付出
